files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -1,97 +1,87 @@
 ---
-# ALL tags must be with dashes (-) instead of underscores (_)
+# NOTE: Use dashes (-) exclusively for tags and underscores (_) exclusively for destinations.
+# submit_request_cpus its called in pulsar and in plain condor only request_cpus
 destinations:
-  interactive_pulsar:
-    runner: pulsar_embedded
-    destination_name_override: "embedded_pulsar_interactive_c{cores}-m{mem}"
-    max_accepted_cores: 24
-    max_accepted_mem: 128
-    max_accepted_gpus: 0
+  ######################
+  # BASIC DESTINATIONS #
+  ######################
+  basic_docker_destination:
     params:
-      tmp_dir: true
       docker_enabled: true
       docker_sudo: false
       docker_net: bridge
       docker_auto_rm: true
       docker_set_user: ""
       require_container: true
-      # submit_request_cpus its called in pulsar and in plain condor only request_cpus
       submit_request_cpus: "{cores}"
       submit_request_memory: "{mem}G"
       outputs_to_working_directory: false
       container_monitor_result: callback
-      submit_requirements: 'GalaxyDockerHack == True'
+      submit_requirements: "GalaxyDockerHack == True"
+
+  basic_singularity_destination:
+    params:
```

Comment:
Is this a destination that jobs can go to, or is it a base to be inherited by other destinations? If this is not a destination that jobs will go to, it can have `abstract: true` so that TPV will never consider it as a candidate destination.